composer.json

```diff
@@ -18,6 +18,7 @@
         "spryker/sales": "^8.5.0 || ^10.0.0 || ^11.0.0",
         "spryker/step-engine": "^3.1.0",
         "spryker/symfony": "^3.0.0",
+        "spryker/transfer": "^3.25.0",
```

Comment:
Dependency `spryker/transfer` is now listed within `require` and `require-dev` while only one is needed. Please check dependencies, because could be that there are more violations. You can do that by executing the following command:

`dev:dependency:find SprykerEco.CrefoPay`

Reply:
Please check again ... if you execute mentioned command you will get errors.

Reply:
Fixed manually, for some reason command return no errors